game_eggs/steamcmd_servers/conan_exiles/egg-conan-exiles.json

```diff
@@ -35,18 +35,20 @@
             "description": "The ID corresponding to the game to download.",
             "env_variable": "SRCDS_APPID",
             "default_value": "443030",
-            "user_viewable": true,
+            "user_viewable": false,
             "user_editable": false,
-            "rules": "required|numeric|digits_between:1,7"
+            "rules": "required|numeric|digits_between:1,7",
```

Comment:
This can just be `required|numeric|in:443030`

Reply:
not really needed, because its only steam AppID and can also be requiered|string|